examples/visual-tests/amp-story/amp-story-shopping.js

```diff
@@ -91,4 +91,20 @@ module.exports = {
     );
     await page.waitForSelector('.i-amphtml-amp-story-shopping-pdp');
   },
+  'displays shopping PDP (product description page) by default when one attachment is on the page':
+    async (page, name) => {
+      const pageID = 'remote-with-product';
+      const url = await page.url();
+      await page.goto(`${url}#page=${pageID}`);
+      await page.waitForSelector(
+        `amp-story-page#${pageID}[active][distance="0"]`
+      );
+      await page.tap(
+        `amp-story-page#${pageID} .i-amphtml-story-inline-page-attachment-chip`
+      );
+      await verifySelectorsVisible(page, name, [
+        '.i-amphtml-story-draggable-drawer-open',
+      ]);
+      await page.waitForSelector('.i-amphtml-amp-story-shopping-pdp');
```

Comment:
Does `.i-amphtml-amp-story-shopping-pdp` always exist somewhere within the shopping attachment? If so, then should we use `verifySelectorsVisible()` as a more useful test?

Reply:
It doesn't build until the attachment is opened. I think `waitForSelector` is okay because of that.

Reply:
Got it, although you can update `waitForSelector` to verify that the PDP is displayed by default instead of solely verifying that the PDP exists somewhere in the DOM:
```
page.waitForSelector('.i-amphtml-amp-story-shopping-pdp', {visible: true})
```

Reply:
🔥 https://puppeteer.github.io/puppeteer/docs/puppeteer.page.waitforselector/
TIL about options on `waitforselector` :) Thank you! 
I updated the other tests with this option.